<?xml version="1.0"?>
<launch>

  <arg name="model" default="$(find mros1_reasoner)/owl/URJCpilot.owl"/>
  <arg name="tomasys" default="[$(find mros1_reasoner)/owl/tomasys.owl,
                               $(find mros1_reasoner)/owl/MROS_ontology.owl,
                               $(find mros1_reasoner)/owl/navigation_domain_ont.owl]"/>

  <arg name="desired_configuration" default="f_normal_mode"/>
  <arg name="nfr_energy" default="0.5"/>
  <arg name="nfr_safety" default="0.5"/>
  <arg name="reasoning_rate" default="2.0"/>
  <arg name="use_reconfigure_srv" default="True"/>
  <arg name="reconfigure_srv_name" default="rosgraph_manipulator_action_server"/>

  <node name="reasoner" pkg="mros1_reasoner" type="mros1_reasoner_node.py" output="screen">
    <param name="model_file" type="string" value="$(arg model)"/>
    <rosparam param="tomasys_file" subst_value="True">$(arg tomasys)</rosparam>
    <param name="desired_configuration" type="string" value="$(arg desired_configuration)"/>
    <param name="nfr_energy" value="$(arg nfr_energy)"/>
    <param name="nfr_safety" value="$(arg nfr_safety)"/>
    <param name="reasoning_rate" value="$(arg reasoning_rate)"/>
    <param name="use_reconfigure_srv" value="$(arg use_reconfigure_srv)"/>
    <param name="reconfigure_srv_name" value="$(arg reconfigure_srv_name)"/>
    
  </node>

</launch>